marlowe-apps/pipe/Main.hs

```diff
@@ -0,0 +1,56 @@
+
+
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+
+module Main
+  ( main
+  ) where
+
+
+import Control.Monad.Except (liftIO, runExceptT)
+import Data.Aeson (eitherDecode, encode)
+import Data.Text (Text)
+import Language.Marlowe.Runtime.App.Parser (getConfigParser)
+import Language.Marlowe.Runtime.App.Transact (handleWithEvents)
+import Observe.Event (addField, hoistEvent, withEvent)
+import Observe.Event.Dynamic (DynamicEventSelector(..))
+import Observe.Event.Render.JSON (DefaultRenderSelectorJSON(defaultRenderSelectorJSON))
+import Observe.Event.Render.JSON.Handle (simpleJsonStderrBackend)
+import Observe.Event.Syntax ((≔))
+
+import qualified Data.ByteString.Lazy.Char8 as LBS8 (getContents, lines, putStrLn, unpack)
+import qualified Options.Applicative as O
+
+
+main :: IO ()
+main =
```

Comment:
This is a cool app, but is this PR the right place to add it?

Reply:
Perhaps not, but it was in `marlowe-lambda`, which is deprecated now.

Reply:
Removed in dd7cbd5b7646c2d03f5ee88274dab8653a21aba8.